crates/wildbg-c/src/lib.rs

```diff
@@ -97,19 +97,18 @@ impl From<&Probabilities> for CProbabilities {
     }
 }
 
-/// When no move is possible, all member variables in all details will be `-1`.
+/// When no move is possible, move_count will be 0.
 ///
-/// If only one checker can be moved once, `detail1` will contain this information,
-/// `detail2`, `detail3` and `detail4` will contain `-1` for both `from` and `to`.
+/// If only one checker can be moved once, `moves[0]` will contain this information,
+/// `moves[1]`, `moves[2]` and `moves[3]` will contain `-1` for both `from` and `to`.
+/// move_count will contain a value between 0 and 4.
 ///
 /// If the same checker is moved twice, this is encoded in two details.
 #[repr(C)]
 #[derive(Default)]
 pub struct CMove {
-    detail1: CMoveDetail,
-    detail2: CMoveDetail,
-    detail3: CMoveDetail,
-    detail4: CMoveDetail,
+    moves: [CMoveDetail; 4],
```

Comment:
I think it's confusing when a `move` consists of `moves`. Maybe a `turn` can have `moves`, or a `move` can have `parts`, `steps`, `details`. I went with `details` in the web API, I think we should align here.

Reply:
Good suggestion. I've changed the array to be called `details`. This also nicely aligns with the actual type name (CMoveDetail) 😄